src/AppBundle/Validator/Constraints/IsValidDeliveryDateValidator.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+
+namespace AppBundle\Validator\Constraints;
+
+use AppBundle\Entity\Restaurant;
+use Carbon\Carbon;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+class IsValidDeliveryDateValidator extends ConstraintValidator
+{
+
+    public function validate($object, Constraint $constraint)
+    {
+        $now = Carbon::now();
```

Comment:
@alexsegura for the life of me I couldn't find a way to inject a date here for unit testing, and it's so easy with Carbon... :/

I don' t have  any other solution for now :/

Reply:
Looks good to me.